Mesh4all/m4a-firmware
# ci: Static-tests resolving the doccheck errors

### Contribution description

PR designated to resolve the errors and warnings from static-tests doccheck,  



### Testing procedure

General folder (m4a-firmware):
```sh
make static-test
```
Specific tool:
```sh
cd dist/tools/doccheck
./check.sh
```
### Issues/PRs references

Related to  #41 #40 
Fix #34
